scripts/static_remote.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -ev
+
+./scripts/run_bitcoin.sh
+./scripts/run_lnd.sh carlakirkcohen/lnd:v0.11.1-beta 0
+./scripts/run_lnd.sh carlakirkcohen/lnd:v0.7.0-beta 1
```

Comment:
did you test without the patch, checking whether v0.7 would disconnect from `master` when a legacy channel existed? AFAIK eclair is even stricter than LND in how it handles unknown required bits.

Reply:
0.7 disconnect because on the unknown feature bit and 0.11 will keep trying to connect because we have a channel open. Tested this out manually on master 👍